File: server/src/main/scala/producer/message_generator/batcher_builder/DefaultBatcherBuilder.scala
package producer.message_generator.batcher_builder

import com.tools.teal.pulsar.ui.producer.v1.producer as pb

case class DefaultBatcherBuilder()

object DefaultBatcherBuilder:
    def fromPb(v: pb.DefaultBatcherBuilder): DefaultBatcherBuilder =
        DefaultBatcherBuilder()

    def toPb(v: DefaultBatcherBuilder): pb.DefaultBatcherBuilder =
        pb.DefaultBatcherBuilder()

File: ui/components/ui/ProducerSession/producer-task/pulsar-producer-config/batcher-builder/default-batcher-builder.ts
import * as pb from '../../../../../../grpc-web/tools/teal/pulsar/ui/producer/v1/producer_pb';

export type DefaultBatcherBuilder = {
  type: 'default-batcher-builder'
};

export function defaultBatcherBuilderFromPb(v: pb.DefaultBatcherBuilder): DefaultBatcherBuilder {
  return { type: 'default-batcher-builder' };
}

export function defaultBatcherBuilderToPb(v: DefaultBatcherBuilder): pb.DefaultBatcherBuilder {
  return new pb.DefaultBatcherBuilder();
}
